Sources/Utilities.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+struct Utilities {
```

Comment:
I would suggest using a different name than `Utilities` (which is both very ambiguous, and tends to be a "catch all" for all kinds of unrelated APIs). Since these utilities are all about encoding strings, how about implementing them in an extension on `String`?